### PR TITLE
Add toggle option for "abstract" field.

### PR DESCRIPTION
**Pull request by ran81**
#33

### CODE REVIEW

File: modules/biblio_example/js/biblio_example.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(function($) {
+
+  Drupal.behaviors.toggleAbstract = {
+    attach: function() {
+      $(".show-abstract").click(function() {
+
+        var bid = $(this).attr('bid');
+
+        $("div .bid_" + bid).slideToggle("fast");
```
Comment (amitaibu) [inline modules/biblio_example/js/biblio_example.js]: `_` => `-`

File: modules/biblio_example/js/biblio_example.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+(function($) {
+
+  Drupal.behaviors.toggleAbstract = {
```
Comment (amitaibu) [inline modules/biblio_example/js/biblio_example.js]: What does it do?

Comment (amitaibu) [inline modules/biblio_example/js/biblio_example.js]: Indentation to the left

**Comment by amitaibu** [comment]
We need to make sure the JS/ CSS can be loaded even if the biblio is cached.

Move JS and CSS to the plugin options.

**Comment by amitaibu** [comment]
https://gist.github.com/amitaibu/6119981
- Add defaults in `biblio_plugin_process`
- Add CSS/ JS in `Biblio::getText()`

**Comment by amitaibu** [comment]
Can you add also a CSS to style it like the original one.

**Comment by ran81** [comment]
Abstract hidden:
close

Abstract shown:
open

File: modules/biblio_example/css/biblio_example.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+a.show-abstract {
+    background: url('../images/scholarbase-sprite.png') no-repeat 0 -399px transparent;
```
Comment (amitaibu) [inline modules/biblio_example/css/biblio_example.css]: Indentation (2 spaces)

File: modules/biblio_example/css/biblio_example.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    padding-left: 12px;
+    text-decoration: none;
+    color: #336599;
+}
+
+a.show-abstract.open {
+    background: url('../images/scholarbase-sprite.png') no-repeat 0 -412px transparent;
```
Comment (amitaibu) [inline modules/biblio_example/css/biblio_example.css]: scholarbase => biblio-example

File: modules/biblio_example/js/biblio_example.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+(function($) {
+
+// Controls the display of the "Abstract" field of a publication.
```
Comment (amitaibu) [inline modules/biblio_example/js/biblio_example.js]: `//`

```
/**
 *
 */
``
```

File: modules/biblio_example/js/biblio_example.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Drupal.behaviors.toggleAbstract = {
+  attach: function() {
+    $(".show-abstract").click(function() {
+
+      var bid = $(this).attr('bid');
+
+      if ($(this).hasClass('open')){
```
Comment (amitaibu) [inline modules/biblio_example/js/biblio_example.js]: `$.toggleClass()`

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**